bbc/simorgh
# Correct hash for chunkhash

Resolves #1175

**Overall change:** Changes the hash in webpack to chunkhash

**Code changes:**

- Changes the hash in webpack to chunkhash. See [WebPack docs](https://webpack.js.org/configuration/output/#output-filename) for why. But briefly, it should have always been this, it is the has of the chunk content rather than the module identifier.

**Test Instructions**

- General regression test.
- Also: `npm run build`. Make a note of the hashes under `build/`.
- Make a change to the application, e.g. `Document.jsx`.
- Run `npm run build` again. The hash of the `main` bundle should be different, but crucially, any vendor bundles should have the same hash as before.

---

~~- [ ] Tests added for new features~~
